### PR TITLE
refactor: remove unnecessary `Box` from into_inner

### PR DESCRIPTION
**Pull request by hamirmahal**

### CODE REVIEW

File: src/decoder/symphonia.rs [hunk 1]
```diff
@@ -48,7 +48,7 @@ impl SymphoniaDecoder {
         }
     }
 
-    pub fn into_inner(self: Box<Self>) -> MediaSourceStream {
```
Comment (hamirmahal) [inline src/decoder/symphonia.rs:51]: `self` doesn't need to be boxed here.

**Comment by dvdsk** [comment]
`mod symphonia` is private to rodio, so this change is possible without changing rodio's API. 

Could you also remove the Box in the caller?

**Comment by hamirmahal** [comment]
Sure! Is 371d963 what you're looking for?

**Comment by dvdsk** [comment]
perfect!

**Pull request merged**